### ISSUE TITLE
[Assistive tech and device support] Focus state isn't visible. (11.09.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43975]
#### VFS team name
Debt Resolution

#### VFS product name
Combined VA Debt Portal

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Focus state isn't visible.

### Issue Details
When viewing a statement for a copay or debt, the link to download a PDF copy of the statement does not have a visible focus indicator when it has keyboard `:focus` and doesn't appear to have the same styling on `:hover` either.

For keyboard-only or keyboard-first users, visible focus indicators are essential for tracking where you are on the page. Without a visible indicator, a keyboard user is left to guess whether or not pressing [Enter] will actually activate the link they're trying to reach.

### Link, screenshot or steps to recreate
1. Logged in as test user 80, navigate to the [June 5 statement](https://staging.va.gov/manage-debt-and-bills/summary/copay-balances/702c94e3-be4a-44f5-be80-f261d4baddfd/detail/statement).
2. Press [Tab] to move through the links on the page.
3. Note that each tab stop has a yellow outline -- until the "Download your June 5, 2021 statement (PDF)" link.

### VA.gov Experience Standard
[Category Number 11, Issue Number 09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.7_AA

---

### Platform Recommendation
I haven't dug into the CSS closely enough to figure out what makes this link different from the others, but I'm wondering if the `p` nested inside the `a` might be causing an issue? That's an unusual pattern that I don't think I spotted anywhere else in my review, and it might be causing some CSS wonkiness too.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.
